doc/examples/applications/plot_text.py

```diff
@@ -0,0 +1,38 @@
+"""
+=========================
+Render text onto an image
+=========================
+
+Scikit-image currently doesn't feature a function that allows you to
+plot/render text onto an image. However, there is a fairly easy workaround
+using the `matplotlib <https://matplotlib.org/>`_ library.
+
+"""
+
+import numpy as np
+import imageio.v3 as iio
+import matplotlib.pyplot as plt
+from matplotlib.figure import figaspect
+
+img = iio.imread("imageio:chelsea.png")
```

Comment:
I can change it if desired. I wrote it like this mostly out of habit.

ImageIO has its own [collection of standard images](https://imageio.readthedocs.io/en/stable/user_guide/standardimages.html) which partly overlaps with the images in `skimage.data`. This makes me wonder if it would be beneficial to merge them, considering that `skimage.io` will eventually be deprecated in favor of using ImageIO directly.

Reply:
I don't have a strong opinion on this but think our version would be less surprising. :wink: 

Well, skimage has not to long ago changed the way we provide the images in `data` using pooch and started hosting new images separately from our repository at https://gitlab.com/scikit-image/data. Considering that imageio is a dependency of skimage I would expect the merging would happen by moving this functionality to imageio? Are you prepared to support that approach? Another, approach might involve only merging the more minimal set of data files that are included in the repository and installation archives.

Reply:
Then I'll change it.

Regarding standard images, we added our own download/caching mechanism about 6 years ago, so I guess we are supporting such a mechanism already. There are still some images left in ImageIO's wheel, but we have plans to remove them ([see here for the discussion)](https://github.com/imageio/imageio/issues/827). 

If this sounds like a useful addition to skimage I suggest we make this a dedicated issue in this repo to discuss the details and to place it in a more prominent place where others can contribute to the discussion. @lagru What do you think?